name-lint.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+LINT="$(grep -e "Pycon UK" -e "pycon UK" -e "pyconUK" -e "PyConUK" \
+        --line-number --recursive . \
+        --include "*.html" --include "*.rst")"
+
+SUCCESS=0
+
+if [[ $LINT ]] ;  then
```

Comment:
I believe you can make the same test here with single brackets making the check shell agnostic =)